numpy/distutils/system_info.py

```diff
@@ -1877,10 +1884,8 @@ class blis_info(blas_info):
 
     def calc_info(self):
         lib_dirs = self.get_lib_dirs()
-        blis_libs = self.get_libs('libraries', self._lib_names)
-        if blis_libs == self._lib_names:
-            blis_libs = self.get_libs('blis_libs', self._lib_names)
```

Comment:
I hadn't realized we already accepted `libraries` elsewhere. I think using this approach with the `if` statement everywhere makes sense to me.

Reply:
Yes, it has been there for some time. But I dislike the above since one could put the same entries in the `libraries` key which would then allow the 2nd key.

In my opinion we should fix a `libraries` preference, and allow the older keywords (as exceptions). Once I have the error handler I will update.